app/serializers/plant_serializer.rb

```diff
@@ -0,0 +1,12 @@
+class PlantSerializer
+  include FastJsonapi::ObjectSerializer
+  attributes :image
+  attributes :name
+  attributes :species
+  attributes :description
+  attributes :light_requirements
+  attributes :water_requirements
+  attributes :when_to_plant
+  attributes :harvest_time
+  attributes :common_pests
```

Comment:
Let's include plant relationships here at the bottom.
`has_many :garden_plants`
 `has_many :gardens, through: :garden_plants`

Reply:
My local branch was behind the remote branch. I made the change by editing the file here on GH,